Sources/Contour/Configuration/IBusConfigurator.cs

```diff
@@ -294,5 +296,12 @@ public interface IBusConfigurator
         /// <param name="endpointBuilder">
         /// </param>
         void UseSubscriptionEndpointBuilder(Func<ISubscriptionEndpointBuilder, ISubscriptionEndpoint> endpointBuilder);
+
+
+        /// <summary>
+        /// Sets excluded incoming message headers
+        /// </summary>
+        /// <param name="excludedHeaders">Excluded headers</param>
+        void SetExcludedIncomingHeaders(IEnumerable<string> excludedHeaders);
```

Comment:
There's no scope for endpoint configuration, this improvement is not a subject to change endpoint configuration